src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -432,7 +432,7 @@ void initializeFeatures() {
      * Package-private so that engine classes can access it, but concrete tool child classes cannot.
      * May be overridden by traversals that require custom initialization of intervals.
      */
-    void initializeIntervals() {
+    protected void initializeIntervals() {
```

Comment:
So we don't necessarily want to expose the `initializeIntervals()` argument to tool authors (mostly because we don't want to worry about tool authors changing the internal user intervals in unsafe ways. Since what you primarily need right here is to maybe mutate the user intervals on this line: `userIntervals = intervalArgumentCollection.getIntervals(sequenceDictionary);` I would consider adding a seperate method to do this.

Add a protected method `transformTraversalIntervals(getIntervals, sequenceDictionary)` that returns a list of intervals and make that what GATKTool sets as `userIntervals`. Then you can add the mutation you need to genomicsDBImport and leave the default behavior untouched without exposing anything dangerously by just making the default behavior return the supplied interval list.

Reply:
Can you revert the change of `initializeIntervals()` to protected?